setup.cfg

```
@@ -89,7 +89,7 @@ dev =
 azure = dvc-azure==2.20.0
 gdrive = dvc-gdrive==2.19.0
 gs = dvc-gs==2.19.1
-hdfs = dvc-hdfs==2.19.0; python_version < '3.11'
+hdfs = dvc-hdfs==2.19.0; python_version < "3.11"
```

```suggestion
hdfs =
    dvc-hdfs==2.19.0; python_version < '3.11'
```

I think this alone should fix the issue. We don't need to change quotes.